documentation/sdn/ovn_bgp_evpn_deployments.md

```diff
@@ -0,0 +1,696 @@
+# OVN BGP Agent in Kolla-Ansible for scalability and cloud interconnect deployments
+
+## Test deployment of OVN BGP Agent in Kolla-Ansible
+This part is a quick how-to for setting up OVN BGP Agent in Kolla-ansible.
+There's an existing [PR](https://review.opendev.org/c/openstack/kolla-ansible/+/891622) in kolla-ansible upstream, however it does not work properly. I created a [PR in kolla-ansible fork](https://github.com/SovereignCloudStack/kolla-ansible/pull/5) as well as in [kolla fork](https://github.com/SovereignCloudStack/kolla/pull/2) that attempts to fix the issues.
+
+I got all containers running, however they do not seem to activate  (see below)
+
+### Steps
+- Get a working test environment i.e. testbed, cloud-in-a-box, [kolla-builder](https://github.com/SovereignCloudStack/kolla-builder)
+
+ - If you use kolla-builder, you need to checkout a dev branch `ovn-bgp-agent-deployment`
+ ```bash
+ git fetch origin ovn-bgp-agent-deployment:ovn-bgp-agent-deployment
+ git checkout ovn-bgp-agent-deployment
```

Comment:
Instead of the above, you can use just 
```bash
 git clone --branch ovn-bgp-agent-deployment https://github.com/SovereignCloudStack/kolla-builder.git
```